CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # App Center SDK for Unity Change Log
 
+## Version 4.3.0
+
+### AppCenter
+
+#### UWP
+
+* **[Fix]** Fix sending pending logs after the first application start.
+
+#### iOS
```

Comment:
Shouldn't we unity ios and android like: #### iOS/Android, as we [did in React Native](https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/CHANGELOG.md#androidios)?